tests/common.py

```diff
@@ -150,29 +150,6 @@ def prepare_nb_system(x, E, lambda_plane_idxs, lambda_offset_idxs, p_scale, cuto
     return params, ref_total_energy, test_potential
 
 
-def prepare_restraints(x, B, precision):
-
-    assert x.ndim == 2
-    N = x.shape[0]
-    # D = x.shape[1]
-
-    atom_idxs = np.arange(N)
-
-    params = np.random.randn(B, 3).astype(np.float64)
-
-    bond_idxs = []
-    for _ in range(B):
-        bond_idxs.append(np.random.choice(atom_idxs, size=2, replace=False))
-    bond_idxs = np.array(bond_idxs, dtype=np.int32)
-
-    lambda_flags = np.random.randint(0, 2, size=(B,)).astype(np.int32)
-
-    custom_restraint = potentials.Restraint(bond_idxs, params, lambda_flags, precision=precision)
```

Comment:
(Note: the test function to be removed was also broken at this line -- the reference to `potentials.Restraint` does not resolve, so the reference to `bonded.restraint` (removed in this PR) would not currently be reached if `prepare_restraints` were called.)